processor/src/operations/crypto_ops.rs

```diff
@@ -103,8 +103,8 @@ where
     ///
     /// To perform the operation we do the following:
     /// 1. Update the leaf node at the specified index in the advice provider with the specified
-    ///    root, and get the Merkle path to this leaf. If `copy` is set to true, we make a copy
-    ///    of the merkle set before updating it.
+    ///    root, and get the Merkle path to this leaf. It will make a copy of the Merkle set before
+    ///    updating it.
```

Comment:
Here, and in other places in this doc comment, we should probably remove references to "leaf node" and replace them with just "node". This is because previously we could update only leaf nodes, but now we can update any node in the tree.

Also, we should replace references to "Merkle set" with "Merkle tree".

Reply:
Makes sense; fixed!